src/conduit/tests/test_check_db_migrated.py

```diff
@@ -0,0 +1,98 @@
+"""Test def check_db_migrated function."""
+
+from conduit import check_db_migrated
+from conduit import main
+from unittest import mock
+
+
+@mock.patch("conduit.alembic")
+def test_skip_when_running_an_alembic_command(alembic):
+    """Pyramid env is bootstrapped when running alembic commands.
+
+    We need to skip checking when this is the case, because then you cannot
+    ever migrate the database.
+    """
+    alembic.context = mock.Mock(spec="config".split())
+    config = None
+    global_config = None
+
+    assert check_db_migrated(config, global_config) is None
+
+
+@mock.patch("conduit.alembic")
+@mock.patch("conduit.EnvironmentContext")
+@mock.patch("conduit.ScriptDirectory")
+@mock.patch("conduit.MigrationContext")
+@mock.patch("conduit.sys")
```

Comment:
Use mocker fixture

Reply:
I did. I really don't see the point. Is there a blog post that shows the benefits?